docs/whats-new/dotnet-docs-mod1.md

```diff
@@ -1,108 +1,123 @@
+- [CA2024: Do not use StreamReader.EndOfStream in async methods](../fundamentals/code-analysis/quality-rules/ca2024.md)
+- [Containerize a .NET app with dotnet publish](../core/containers/sdk-publish.md)
+- [Distributed tracing in System.Net libraries](../fundamentals/networking/telemetry/tracing.md)
+- [Keyed DI support in `IHttpClientFactory`](../core/extensions/httpclient-factory-keyed-di.md)
+- [MSTEST0038: Don't use 'Assert.AreSame' or 'Assert.AreNotSame' with value types](../core/testing/mstest-analyzers/mstest0038.md)
+- [MSTEST0039: Use newer 'Assert.Throws' methods](../core/testing/mstest-analyzers/mstest0039.md)
+- [MSTEST0040: Do not assert inside 'async void' contexts](../core/testing/mstest-analyzers/mstest0040.md)
+- [NETSDK1032: RuntimeIdentifier and PlatformTarget must be compatible](../core/tools/sdk-errors/netsdk1032.md)
+- [NETSDK1144: Optimizing assemblies for size failed](../core/tools/sdk-errors/netsdk1144.md)
+- [Preview APIs](../fundamentals/runtime-libraries/preview-apis.md)
+- [QUIC configuration options](../fundamentals/networking/quic/quic-options.md)
+- [Serialization extension methods on HttpClient](../standard/serialization/system-text-json/httpclient-extensions.md)
+- [SYSLIB0058: Certain SslStream properties are obsolete](../fundamentals/syslib-diagnostics/syslib0058.md)
+- [SYSLIB0059: SystemEvents.EventsThreadShutdown callbacks aren't run before the process exits](../fundamentals/syslib-diagnostics/syslib0059.md)
+- [SYSLIB0060: Rfc2898DeriveBytes constructors are obsolete](../fundamentals/syslib-diagnostics/syslib0060.md)
 
 ### Updated articles
 
-- [HttpWebRequest to HttpClient migration guide](../fundamentals/networking/http/httpclient-migrate-from-httpwebrequest.md) - Fix build suggestions
-- [Install .NET on macOS](../core/install/macos.md) - Rewrite install on macOS article
+- [Creating metrics](../core/diagnostics/metrics-instrumentation.md) - [diagnostics] Add InstrumentAdvice details to instrumentation doc
+- [dotnet-coverage code coverage utility](../core/additional-tools/dotnet-coverage.md) - Update dotnet-coverage docs. Adding uninstrument command
+- [Microsoft.Testing.Platform and VSTest comparison](../core/testing/unit-testing-platform-vs-vstest.md) - Add first level info for migration off of VSTest
+- [Tutorial: Containerize a .NET app](../core/docker/build-container.md) - Address issues related to .NET containers
+
+## C# language
+
+### New articles
+
+- [Custom attributes that generate flags or options in the Intermediate Language (IL) output](../csharp/language-reference/attributes/pseudo-attributes.md)
+
+## Visual Basic language
+
+### Updated articles
+
+- [Generic types in Visual Basic (Visual Basic)](../visual-basic/programming-guide/language-features/data-types/generic-types.md) - Add VB new features
+
+## AI in .NET
+
+### New articles
+
+- [Create a minimal AI assistant using .NET](../ai/quickstarts/quickstart-assistants.md)
```

Comment:
Ahh, here it is @BillWagner. Could we make this the first section in the what's new?

Reply:
Yes! 

It's already in the algorithm.  All we need to do is change the order in the `.whatsnew.json` file. 

I'll make a PR tomorrow (after merging this) to update the config and move the existing sections. That way, future diffs will be easier to understand.

I'll suggest putting the AI section after the security and breaking changes, but before everything else. Do you agree, or should AI be above those as well?

Reply:
That sounds good, thanks.

Reply:
Implemented in #44677